packages/seacas/libraries/ioss/src/Ioss_Region.C

```diff
@@ -3055,4 +3105,28 @@ namespace Ioss {
       topologyObserver->reset_topology_modification();
     }
   }
+
+  std::string Region::get_group_name() const
+  {
+    return get_database()->get_group_name();
+  }
+
+  std::tuple<std::string, int, double> Region::locate_db_state(const double targetTime)
```

Comment:
Does this only work for the cases where the exodus file contains groups, or will it return the "group" correspoinding to a set of files `-s000X` storing topology changed data...

Reply:
Yeah, this is really only for group files but I might be able to modify it to scan the multi-file case....the return string would have to be the database name, I guess

Reply:
I guess it depends on how complete and broad we are trying to make the capability.  I don't know what the clients will expect -- I could see them wanting to use the same interface for all database types (that support restart/output) whether they use groups or different files, or a different method all together....  If it doesn't look too difficult then might be worthwhile...